php7-fpm/Dockerfile

```diff
@@ -12,8 +12,9 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 RUN composer --version
 
 # Set timezone
-RUN rm /etc/localtime
-RUN ln -s /usr/share/zoneinfo/Europe/Paris /etc/localtime
+ENV TZ=Europe/Paris
```

Comment:
Hi @carlosas, thanks for this PR!
Could you put this line to the [`.env.dist` file](https://github.com/maxpou/docker-symfony/blob/master/.env.dist)?